scripts/dts/edtlib.py

```diff
@@ -101,8 +101,8 @@ def __init__(self, dts, bindings_dir):
           Path to device tree .dts file
 
         bindings_dir:
```

Comment:
might want to rename this to `bindings_dirs`

Reply:
I actually had this changed all through-out (in both files).  But I thought it made the changes look worse than they actually are.

Reply:
IE: since it was a regression I wanted to keep it simple